src/tracker/psmove_tracker.c

```diff
@@ -414,7 +436,10 @@ PSMoveTracker *psmove_tracker_new() {
         camera = 0;
     }
 #endif
-
+#if defined(_WIN32) && defined(CAMERA_CONTROL_USE_PS3EYE_DRIVER)
+    printf("\nDetected PS3EYEDriver in Windows. You may have to `SET %s=2` (or =1).\n",
```

Comment:
Ok, works for me - as long as we're on the same page that this should eventually be fixed (probably PS3EYEDriver can be fixed to not just crash'n'burn when it opens the wrong device).